utils/fixed_decimal/src/decimal.rs

```diff
@@ -350,65 +351,103 @@ impl FixedDecimal {
     /// let mut dec = FixedDecimal::from(42);
     /// assert_eq!("42", dec.to_string());
     ///
-    /// dec.pad_or_truncate_left(2);
+    /// dec.padded_left(3);
```

Comment:
Thought: I think most users would expect the argument to be "number of integer digits", rather than "power of 10 of the leftmost digit".  We should either keep the proposed behavior and add more documentation language about it (rather than hiding it only in an example), or adjust the behavior of the function.

Reply:
yeah I'll change it to work on number of digits, I like that